### PR TITLE
Saherwasim.md

### PR DESCRIPTION
**Pull request by swasim**
[Rawgit link](https://rawgit.com/swasim/SaherWasim.github.io/saherwasim/#!pages/vi/profiles/saherwasim.md)

### CODE REVIEW

**Comment by xyb994** [comment]
@swasim Your LinkedIn link is not working. Adding `https://` might help, it looks like the url is also wrong.

File: pages/vi/profiles/saherwasim.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+*******************
+
+Living in the heart of San Francisco, where, I have now been a frontend developer for 4 years! 
+Well, when you don't see me solving problems, then you might just find me saving all the animals 
+in the world! I LOVE DOGS!
+## Email
+>saherwasim1@gmail.com
+
+## Linkedin
+>[Check out my Linkedin profile!](linkedin.com/in/saher-wasim—the--coder)
```
Comment (yaoxing1990) [inline pages/vi/profiles/saherwasim.md]: Your Linkedin file is broken

**Comment by xyb994** [comment]
@swasim Are you still working on this?

**Comment by kylemathias** [comment]
Closing for now. Reopen if you deiced to continue.

**Pull request closed without merging**